app/controllers/cuisines_controller.rb

```diff
@@ -0,0 +1,5 @@
+class CuisinesController < ApplicationController
+  def index
+    render :json => Cuisine.all.to_json(:except => [:created_at, :updated_at, :id])
```

Comment:
minor general comment. 
Since ruby 1.9 (which is quite a long time ago)
you passing/setting key-value with a symbol as the key type you don't have to use the hash-rocket (`=>`) way. instead you can do it like so:

```ruby
render json: Cuisine.all.to_json(except: [:created_at, :updated_at, :id])
```

I think it's much more readable and shorter as well.

However, if you are using a String as your key type than you will need to use hash-rockets:
```ruby
{"my_string_key" => "value", symbol_key: "value"}
```